README.md

```diff
@@ -7,17 +7,7 @@ Notes on running the app locally
 
 1) Run npm install in both backend and frontend folders.
 
-2) Add the following change to frontend/src/app/modules/api/client-config.js:
-
-    ```
-    --- a/frontend/src/app/modules/api/client-config.js
-    +++ b/frontend/src/app/modules/api/client-config.js
-    @@ -1,4 +1,4 @@
-     export const config = {
-    -    baseURL: `${window.location.protocol}//${window.location.hostname}/api`,
-    +    baseURL: `${window.location.protocol}//${window.location.hostname}:3000`,
-         timeout: 5000
-     }
-    ```
+2) Copy `frontend/.env.local.example` to `frontend/.env` 
+(does not apply in case of runnig in docker)
```

Comment:
running

Reply:
Done [here](https://github.com/salsita/foosball-rating/pull/95/commits/0a92aa2b8d648b6df6e9301cec83602f84b58d28#diff-04c6e90faac2675aa89e2176d2eec7d8), sorry about that